bench/process/process.jq

```diff
@@ -0,0 +1,54 @@
+##
+## Aggregation
+##
+def mean:     if length == 0
+                then 0
+                else reduce .[] as $n (0; . + $n) / length end;
+def pow2:     . * .;
+def variance: . | mean as $mean | map_values(. - $mean | pow2) | mean;
+def stddev:   . | (variance ?) | sqrt;
+
+def samples_to_variable (n):
```

Comment:
May at comments to the more elaborate functions? Specially for the ones that don't know jq to well, like me ;)